### PR TITLE
COP-1955 Tab Component

### PR DESCRIPTION
**Pull request by JimmySunDance**

### CODE REVIEW

File: packages/react/src/Components/Header.jsx [hunk 1]
```diff
@@ -9,21 +9,27 @@ const StyledHeader = style.header`
 
 export default function Header() {
   return (
-    <StyledHeader
```
Comment (Ali-Baker) [inline packages/react/src/Components/Header.jsx:12]: You can remove `StyledHeader` 👆 L5

File: packages/react/src/Components/Header.jsx [hunk 2]
```diff
@@ -9,21 +9,27 @@ const StyledHeader = style.header`
...
-      className="govuk-header "
-      role="banner"
-      data-module="govuk-header"
-    >
+    <header className="govuk-header " role="banner" data-module="govuk-header">
       <div className="govuk-header__container govuk-width-container">
+        <div className="govuk-header__logo">
+          <a href="/" className="govuk-header__link govuk-header__link--homepage">
+            <span className="govuk-header__logotype">
+              <svg aria-hidden="true" focusable="false" className="govuk-header__logotype-crown" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 132 97" height="30" width="36">
```
Comment (Ali-Baker) [inline packages/react/src/Components/Header.jsx:17]: Maybe we can save the svg as seperate file?

**Comment by Ali-Baker** [comment]
Looks good so far James. 👏

File: packages/react/src/Components/Header.jsx (hunk 2 above)
Comment (JimmySunDance) [reply packages/react/src/Components/Header.jsx:17]: Image removed as per design prototype.

**Comment by Ali-Baker** [comment]
You can now rebase for the tests to pass.

File: packages/react/package.json [hunk 3]
```diff
@@ -6,7 +6,8 @@
     "govuk-frontend": "^3.8.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "^3.4.3"
+    "react-scripts": "^3.4.3",
+    "styled-components": "^5.2.0"
```
Comment (Ali-Baker) [inline packages/react/package.json:10]: We don't need this package anymore

```suggestion
    "react-scripts": "^3.4.3"
```

**Review by Ali-Baker — CHANGES_REQUESTED** [review]
Need rebase

File: packages/react/src/index.jsx [hunk 4]
```diff
@@ -2,13 +2,16 @@
...
 import ReactDOM from 'react-dom';
 import { initAll } from 'govuk-frontend';
+import 'govuk-frontend/govuk/all.scss';
 
 // Local imports
-import './index.css';
 import App from './App';
 
-initAll();
+document.body.className = document.body.className
```
Comment (Ali-Baker) [inline packages/react/src/index.jsx:10]: I think it might be useful to add comment to explain why we need this.

File: packages/react/src/App.jsx [hunk 5]
```diff
@@ -50,7 +49,6 @@ const App = () => {
       <ImageProvider value={fullpage}>
         <PageBody />
       </ImageProvider>
-      <Footer />
```
Comment (Ali-Baker) [inline packages/react/src/App.jsx:53]: Are we getting rid of the Footer? if so maybe we cal remove Footer.jsx

File: packages/react/src/Components/PageBody.jsx [hunk 6]
```diff
@@ -4,17 +4,21 @@ import React from 'react';
...
 
 const PageBody = () => {
   return (
     <div className="govuk-width-container">
-      <main className="govuk-main-wrapper" id="main-content" role="main">
-        <MatchValue />
-        <PhotoHeaders />
-        <PhotoColumn />
-        <InfoTable />
+      <main className="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
```
Comment (Ali-Baker) [inline packages/react/src/Components/PageBody.jsx:12]: I don't think we nee id?

File: packages/react/src/Components/MRZTable.jsx [hunk 7]
```diff
@@ -0,0 +1,45 @@
+// Global imports
```
Comment (Ali-Baker) [inline packages/react/src/Components/MRZTable.jsx:1]: 👏

File: packages/react/src/Components/MRZPanel.jsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+// Local imports
+import MRZTable from './MRZTable';
+
+const MRZPanel = ({ isActive }) => {
+
+  let displayTab = isActive ? "govuk-tabs__panel" : "govuk-tabs__panel govuk-tabs__panel--hidden";
+
+  return (
+    <div 
+        className={ displayTab }
```
Comment (Ali-Baker) [inline packages/react/src/Components/MRZPanel.jsx:13]:
```suggestion
        className={`
          govuk-tabs__panel
          ${isActive ? 'govuk-tabs__panel--hidden' : ''}
        `}
```

File: packages/react/src/Components/MRZPanel.jsx [hunk 9]
```diff
@@ -0,0 +1,30 @@
+// Global imports
+import React from 'react';
+
+// Local imports
+import MRZTable from './MRZTable';
+
+const MRZPanel = ({ isActive }) => {
+
+  let displayTab = isActive ? "govuk-tabs__panel" : "govuk-tabs__panel govuk-tabs__panel--hidden";
```
Comment (Ali-Baker) [inline packages/react/src/Components/MRZPanel.jsx:9]:
```suggestion

```

File: packages/react/src/Components/MRZPanel.jsx [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+
+  return (
+    <div 
+        className={ displayTab }
+        role="tabpanel" 
+        aria-labelledby="mrz-data" 
+        id="mrz-data"
```
Comment (Ali-Baker) [inline packages/react/src/Components/MRZPanel.jsx:16]:
```suggestion

```

File: packages/react/src/Components/LiveImage.jsx [hunk 11]
```diff
@@ -38,7 +38,6 @@ const LiveImage = () => {
           ref={canvasRef}
           width={CAPTURE_OPTIONS.video.width}
           height={CAPTURE_OPTIONS.video.height}
-          style={{ borderRadius: '10px 10px 0 0' }}
```
Comment (Ali-Baker) [inline packages/react/src/Components/LiveImage.jsx:41]:
```suggestion

```

File: packages/react/src/Components/InfoTabs.jsx [hunk 12]
```diff
@@ -0,0 +1,61 @@
+// Global imports
+import React, { useState } from 'react';
+
+// Local imports
+import MRZPanel from './MRZPanel';
+import ImagePanel from './ImagePanel';
+
+const InfoTabs = () => {
+
+  const [stateTab, setStateTab] = useState({
```
Comment (Ali-Baker) [inline packages/react/src/Components/InfoTabs.jsx:10]:
```suggestion
  const [active, setActive] = useState({
```

File: packages/react/src/Components/InfoTabs.jsx [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+
+  const clickedTab = (panel) => {
+    return panel ? "govuk-tabs__list-item govuk-tabs__list-item--selected" : "govuk-tabs__list-item";
+  }
+
+  return (
+		<div className="govuk-tabs" data-module="govuk-tabs">
+      <ul className="govuk-tabs__list" role="tablist">
+        <li
+          className={ clickedTab(stateTab.imagePanel) }
```
Comment (Ali-Baker) [inline packages/react/src/Components/InfoTabs.jsx:23]: This 👇 or something similar might be cleaner.

```suggestion
          className={`
            govuk-tabs__list-item 
            ${ active.imagePanel ? 'govuk-tabs__list-item--selected' : ''}
          `}
```

File: packages/react/src/Components/InfoTabs.jsx [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+              imagePanel: true,
+              MRZPanel: false,
+              })
+          }
+          >
+            Images to compare
+          </a>
+        </li>
+        <li 
+          className={ clickedTab(stateTab.MRZPanel) }
```
Comment (Ali-Baker) [inline packages/react/src/Components/InfoTabs.jsx:39]: Same here look at 👆. You could do something similar.

File: packages/react/src/Components/ImagePanel.jsx [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+import PhotoHeaders from './PhotoHeaders'
+
+
+const ImagePanel = ({ isActive }) => {
+
+  let displayTab = isActive ? "govuk-tabs__panel" : "govuk-tabs__panel govuk-tabs__panel--hidden";
+
+  return (
+    <div 
+      className={ displayTab }
```
Comment (Ali-Baker) [inline packages/react/src/Components/ImagePanel.jsx:17]: You could use `tabs__panel ${isActive} ? '' : 'govuk-tabs__panel--hidden'`

File: packages/react/src/Components/ImagePanel.jsx [hunk 16]
```diff
@@ -0,0 +1,39 @@
...
+      role="tabpanel" 
+      aria-labelledby="image-data" 
+      id="image-data"
```
Comment (Ali-Baker) [inline packages/react/src/Components/ImagePanel.jsx:20]:
```suggestion

```

File: packages/react/src/Components/PhotoColumn.jsx [hunk 17]
```diff
@@ -8,10 +8,32 @@ import ScanImage from './ScanImage';
 
 const PhotoColumn = () => {
   return (
-    <div className="govuk-grid-row">
-      <ChipImage />
-      <ScanImage />
-      <LiveImage />
+    <div class="govuk-tabs" data-module="govuk-tabs">
```
Comment (Ali-Baker) [inline packages/react/src/Components/PhotoColumn.jsx:11]:
```suggestion
<div className="govuk-tabs" data-module="govuk-tabs">
```

**Review by Ali-Baker — CHANGES_REQUESTED** [review]

**Comment by Ali-Baker** [comment]
Well done 👏  James. This is a big piece of work. Let's try to finish it before the end of the sprint.

File: packages/react/src/App.jsx (hunk 5 above)
Comment (JimmySunDance) [reply packages/react/src/App.jsx:53]: Footer is being removed

File: packages/react/src/Components/MRZPanel.jsx (hunk 9 above)
Comment (JimmySunDance) [reply packages/react/src/Components/MRZPanel.jsx:9]: changes made

**Comment by Ali-Baker** [comment]
https://github.com/UKHomeOffice/ide-controller/pull/33

**Pull request closed without merging**